addons/event/tests/common.py

```diff
@@ -100,9 +100,11 @@ def setUpClass(cls):
         cls.event_0.company_id.write({'country_id': cls.env.ref('base.be').id})
 
     @classmethod
-    def _create_registrations(cls, event, reg_count):
+    def _create_registrations(cls, event, reg_count, create_date=False):
```

Comment:
Do you actually use this new parameter anywhere?

Reply:
No, we don't use this parameter, i will remove it.